cpp/ql/src/Likely Bugs/UnderspecifiedFunctions/TooManyArguments.c

```diff
@@ -0,0 +1,9 @@
+
+void one_argument(int x);
+
+void calls() {
+	
+	one_argument(1); // GOOD: `one_argument` will accept and use the argument
+	
+	one_argument(1, 2); // BAD: `one_argument` will use the first argument but ignore the second
```

Comment:
This example doesn't compile because your declaration of `one_argument` has a prototype.